octomap/src/graph2tree.cpp

```diff
@@ -205,6 +209,7 @@ int main(int argc, char** argv) {
   }
 
   // transform pointclouds first, so we can directly operate on them later
+  if (!dontTransformNodes)
```

Comment:
Could you add indentation and braces to the condition, to make the scope more clear?